src/components/pages/home/index.tsx

```diff
@@ -18,7 +18,9 @@ import InProgressCollection from './in-progress-collection'
 import Jumbotron from './jumbotron'
 import LevelUpCTA from '../../survey/level-up-cta'
 
-const Home: FunctionComponent = () => {
+const Home: FunctionComponent<any> = ({sections}) => {
+  const [featureCallOut] = sections
```

Comment:
suggestion:  think this array aliasing approach is probably not great. I'd consider using named object, similar to `homePageData`

`sections` would be an object and not an array so the names would stay consistent and not "well, the first one in the array is always X and the 7th is Y" situation.